packages/adapter/src/lib/adapter/adapter.ts

```diff
@@ -664,7 +664,7 @@ export class AdapterClass extends EventEmitter {
     protected version?: string;
     protected kill?: () => Promise<void>;
     protected processLog?: (msg: any) => void;
-    protected requireLog?: (_isActive: boolean) => void;
+    protected requireLog?: (_isActive: boolean, options?: any) => void;
```

Comment:
In many other places we have "Unknown" instead of "any" for such options, right @foxriver76 ?

Reply:
does we really have an options attribute which can be passed to `setState`? https://github.com/ioBroker/ioBroker.js-controller/blob/b69925fac26f6faf97ffc3a780b186865bf99071/packages/db-states-redis/src/lib/states/statesInRedisClient.ts#L702

Reply:
if you need iuser rights to be checked you need to use "setState on "adapter instance" because the user checks aredone there ... the States DB is NOT doing any user checks (because do not have the object data)